newIDE/app/src/ProjectManager/index.js

```diff
@@ -760,6 +812,8 @@ export default class ProjectManager extends React.Component<Props, State> {
                     onDelete={() =>
                       this.props.onDeleteExternalEvents(externalEvents)
                     }
+                    typeAddedable={'external-events'}
```

Comment:
onAdd is fine, but typeAddedable is now useless. Instead, pass the label that should be shown for the item:

```
addLabel={"Add a new external event"}
```